wiki/Installation.md

```diff
@@ -0,0 +1,36 @@
+# Swift Package Manager
```

Comment:
We should have someone newer to SPM test that this makes sense.

Reply:
Made a card to do this: https://github.com/wwt/Workflow/projects/1#card-61988366